functors task (1)/student/include/impl.h

```diff
@@ -8,6 +8,11 @@
  */
 struct BeerOrganizer
 {
+	BeerBrand operator() () {
+		static int band = 0;
+		band < 6 && band > 0 ? band++ : band =  1;
```

Comment:
1. Please correct your names, there is a typo
2. **Don't use magic numbers!**
3. Consider plain if